frontend/src/app/processes/processes.component.ts

```diff
@@ -134,6 +151,7 @@ export class ProcessesComponent implements OnInit, OnDestroy {
   public reset(){
     this.selectedApproach = 'None';
     this.selectedMethod = 'None';
+    this.selectedGraphType = 'None';
```

Comment:
```suggestion
    this.selectedApproach = '';
    this.selectedMethod = '';
    this.selectedGraphType = '';
```
I forgot to adapt the reset value of approach and method to the empty default values in the other PR. This should prevent sending unnecessary parameters to the backend after a reset.